TROUBLESHOOTING.md

```diff
@@ -1,5 +1,75 @@
 # Troubleshooting
 
+## ⚠️ Known Issue: "You may not include more than 128 tools in your request"
```

Comment:
this is a VS Code specific limit.  Other Agents have different limits (some even lower); could you tweak the title to mention VS Code?